src/containers/access/src/constants.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+const CONSENT_FORM :'Consent Form' = 'Consent Form';
+const CPHS_REVIEW :'CPHS Review' = 'CPHS Review';
+const DATA_RELEASE_LETTER :'Data Release Letter' = 'Data Release Letter';
+const DATA_SECURITY_LETTER :'Data Security Letter' = 'Data Security Letter';
+const DATA_USE_AGREEMENT :'Data Use Agreement' = 'Data Use Agreement';
+const DEPARTMENT_SUPPORT_LETTER :'Department Support Letter' = 'Department Support Letter';
+const IRB_APPROVAL :'IRB Approval' = 'IRB Approval';
+const PARTICIPANTS_BOR :"Participant's Bill of Rights" = "Participant's Bill of Rights";
```

Comment:
No change required, just a thought. These will eventually need to be tied to the specific type of access request, and will probably need to be managed.